tests/bud.bats

```diff
@@ -33,6 +33,36 @@ load helpers
   rm -rf ${TESTSDIR}/bud/use-layers/mount
 }
 
+@test "bud with --rm flag" {
+  buildah bud --signature-policy ${TESTSDIR}/policy.json --layers -t test1 ${TESTSDIR}/bud/use-layers
+  ctrs1=$(buildah containers | grep "" -c)
+  [[ "$ctrs1" == "1" ]]
+
+  buildah bud --signature-policy ${TESTSDIR}/policy.json --rm=false --layers -t test2 ${TESTSDIR}/bud/use-layers
+  ctrs2=$(buildah containers | grep "" -c)
+  [[ "$ctrs2" == "4" ]]
+
+  buildah rm -a
+  buildah rmi -a
+}
+
+@test "bud with --force-rm flag" {
+  run buildah bud --signature-policy ${TESTSDIR}/policy.json --force-rm --layers -t test1 -f Dockerfile.fail-case ${TESTSDIR}/bud/use-layers
+  echo "$output"
+  [ "$status" -ne 0 ]
+  ctrs1=$(buildah containers | grep "" -c)
```

Comment:
nit: Weird way to count lines.
`wc -l`
Would make more sense to me.

Reply:
Using wc instead.